packages/block-library/src/index.js

```diff
@@ -38,6 +38,7 @@ import * as navigation from './navigation';
 import * as navigationLink from './navigation-link';
 import * as latestComments from './latest-comments';
 import * as latestPosts from './latest-posts';
+import * as loginOut from './loginout';
```

Comment:
That would probably read better as `logInOut`.

Reply:
Makes sense... PR in https://github.com/WordPress/gutenberg/pull/29979 👍